src/renderers/shaders/ShaderLib/cube_vert.glsl.js

```diff
@@ -11,6 +11,8 @@ void main() {
 	#include <project_vertex>
 
 	gl_Position.z = gl_Position.w; // set z to camera.far
+	// important because otherwise the cube used for rendering the cubeMap might
+	// get clipped
```

Comment:
This file has nothing to do with this PR. I think it should be removed from the PR.